ckanext/unaids/theme/templates/request/show.html

```diff
@@ -1,19 +1,4 @@
-{% extends "request/page.html" %}
-
-{% block subtitle %}{{ _('Organization membership request') }}{% endblock %}
-
-{% block secondary_content %}
-{% asset "request_js/request_js" %}
-
-  <section class="module module-narrow">
-    <h2 class="module-heading"><i class="icon-large icon-info-sign"></i> {{ _('Organization membership request') }}</h2>
-    <div class="module-content">
-      <p>
-        {% trans %}This shows a given membership request. Only admins can approve, modify or reject requests.{% endtrans %}
-      </p>
-    </div>
-  </section>
-{% endblock %}
+{% extends "page.html" %}
```

Comment:
this should say `{% ckan_extends %}` so that the template engine replaces the `primary_content` block in the `ytp-request/templates/request/show.html`.

You can read a little more about it here: https://docs.ckan.org/en/2.9/theming/templates.html#extending-templates-with-ckan-extends